manifests/vrrp/sync_group.pp

```diff
@@ -16,19 +16,23 @@
 # $notify_script::         Define the notify script.
 #                          Default: undef.
 #
+# $notify_script_master_rx_lower_pri    Define the notify master_rx_lower_pri script.
+#                                       Default: undef.
+#
 # $smtp_alert::            Send email on status change (Boolean)
 #                          Default: undef.
 #
 #
 define keepalived::vrrp::sync_group (
   $group,
-  $notify_script_master    = undef,
-  $notify_script_backup    = undef,
-  $notify_script_fault     = undef,
-  $notify_script           = undef,
-  $smtp_alert              = undef,
-  $nopreempt               = undef,
-  Boolean $global_tracking = false,
+  $notify_script_master              = undef,
+  $notify_script_backup              = undef,
+  $notify_script_fault               = undef,
+  $notify_script                     = undef,
+  $notify_script_master_rx_lower_pri = undef,
```

Comment:
please add a datatype for the new parameter.

Reply:
See above.